packages/flutter/test/widgets/gesture_disambiguation.dart

```diff
@@ -0,0 +1,45 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter/gestures.dart';
+
+void main() {
+  testWidgets('onTap detection with canceled pointer and a drag listener', (WidgetTester tester) async {
+    int detector1TapCount = 0;
+    int detector2TapCount = 0;
+
+    final Widget widget = new GestureDetector(
+        child: new Column(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: <Widget>[
+              new GestureDetector(
+                  onTap: () => detector1TapCount++,
+                  behavior: HitTestBehavior.opaque,
+                  child: const SizedBox(width: 200.0, height: 200.0),
+              ),
+              new GestureDetector(
+                onTap: () => detector2TapCount++,
+                behavior: HitTestBehavior.opaque,
+                child: const SizedBox(width: 200.0, height: 200.0)
+              )
+            ],
+          ));
```

Comment:
the last parenthesis should be on its own line. the rule is if there's a newline after an open paren, there should be a newline before the close paren.

Also the indentation should be such that the close paren lines up with the first non-space character of the line that finished with the open paren.

Reply:
done